x-pack/plugins/synthetics/public/apps/synthetics/components/common/components/monitor_status.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { EuiBadge, EuiDescriptionList, EuiLoadingSpinner } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { EncryptedSyntheticsMonitor } from '../../../../../../common/runtime_types';
+
+export const MonitorStatus = ({
```

Comment:
++ on a separate file, I was planning on breaking these up before merging the original PR.

Do you think as a follow-up we could make a `flyout` folder and colocate all these files in there? I think `MonitorStatus` is a fairly generic component name, but it's not a very generic component as it nests a description list with some opinionated style choices. If we keep it here it might be a good idea to extract the EUI layout code to a separate component.